File: archives/bufferbloat/p_bb_amr_fb_matlab/ns2/drop/drop_plot.py
#!/usr/bin/python

import scipy.io
import matplotlib
from pylab import *
import numpy as np
import matplotlib.ticker as plticker

data = scipy.io.loadmat('matlab.mat')

tocq = data['tocq']
ocq = data['ocq']
tred = data['tred']
red = data['red']

font = {'size': 18, 'family':'serif'}
matplotlib.rc('font',**font)
matplotlib.rc('text', usetex=True)

f1=figure()
plot(tocq,ocq,'k',label='FB-OCQ',  linewidth=2.0)
plot(tred,red,'k--',label='RED',  linewidth=2.0)
legend(loc='best', fontsize=16)
xlabel('Time (sec)')
ylabel('Dropping Probability')

show()

File: archives/bufferbloat/p_bb_amr_fb_matlab/ns2/drop/drop_plot.m
clear;
clc;
load matlab.mat

plot(tocq,ocq,'k','LineWidth',2)
hold on
plot(tred,red,'k--','LineWidth',2)
hold off

xlabel('Time (sec)','FontSize',14)
ylabel('Dropping Probability','FontSize',14)
set(gca, 'XLim',[0 60])
set(gca,'XTick',(0:10:60))
set(gca, 'fontsize',14)
h_legend=legend('OCQ','RED','Location','Best');
set(h_legend,'FontSize',14);